packages/core/src/cloud/LlamaCloudRetriever.ts

```diff
@@ -1,28 +1,37 @@
-import type { PlatformApi, PlatformApiClient } from "@llamaindex/cloud";
+import type { RetrievalParams, TextNodeWithScore } from "@llamaindex/cloud";
+import * as PlatformApiTypes from "@llamaindex/cloud";
 import type { NodeWithScore } from "../Node.js";
 import { ObjectType, jsonToNode } from "../Node.js";
 import type { BaseRetriever, RetrieveParams } from "../Retriever.js";
 import { Settings } from "../Settings.js";
 import { wrapEventCaller } from "../internal/context/EventCaller.js";
-import type { ClientParams, CloudConstructorParams } from "./types.js";
+import type {
+  ClientParams,
+  CloudConstructorParams,
+  ObjectToCamel,
+} from "./types.js";
 import { DEFAULT_PROJECT_NAME } from "./types.js";
 import { getClient } from "./utils.js";
+
 export type CloudRetrieveParams = Omit<
-  PlatformApi.RetrievalParams,
-  "query" | "searchFilters" | "pipelineId" | "className"
```

Comment:
This sounds not right, it's safe in type but not in runtime, you have to convert it to camel case